server/chat-commands/admin.ts

```diff
@@ -131,10 +131,11 @@ export const commands: ChatCommands = {
 		`/changerankuhtml [rank], [name], [message] - Changes the message previously shown with /addrankuhtml [rank], [name]. Requires: * # &`,
 	],
 
-	addline(target, room, user) {
+	showraw(target, room, user) {
```

Comment:
Still haven't been able to think of a good name. I don't like `showraw` because `raw` currently means "HTML" in PS's codebase.